### PR TITLE
Added chitin and biosilicified chitin chestplates, XL variants and recipes

### PR DESCRIPTION
**Pull request by angryisosceles**
#### Summary
Content "Added chitin and biosilicified chitin chestplates"

#### Purpose of change
Chitin armor has individual arm, leg, head, hand, and foot pieces, but the only torso option covers both the chest and legs.

#### Describe the solution
Added chitin and biosilicified chitin chestplates, XL variants, and recipes. Chestplates are generally congruent with the already existing chitin armor pieces, and are slightly lighter, smaller, and easier to craft than the chitin suit.

#### Testing
Spawned in items and tested crafting recipes with no issues

### CODE REVIEW

**Comment by QuintusAquila** [comment]
It might be better to call it a cuirass since it doesn't only protect the chest with 90% coverage.

**Comment by andrei8l** [comment]
66 chunks of chitin weigh 5.87 kg but produce an armor piece that only weighs 1.7 kg? Where does the rest go and how are you paring down the chunks with just an awl and a pocket knife?

I realize your recipes match the existing ones for chitin armors, which IMO should have been removed along with bone armor. I really doubt you can make a lamellar armor from random scraps of unspecified chitin.

**Comment by Venera3** [comment]
Re: using random chitin pieces, if you'd be prepared to tackle it we could use a larger, heavier variant that's used for armor crafting and only drops from the more armored critters - something like "chitin plates", harvested in single-digit amounts from larger terrestrial insects only. That would also mean that you're not set up to craft all the armor in the world after killing a frigate fly or whatever nonthreatening critter caught your eye.

**Comment by Winterous** [comment]
Shouldn't the recipe include string / whatever else the other recipes use? It needs SOMETHING to hold it together.

File: data/json/items/armor/torso_armor.json [hunk 1]
```diff
@@ -508,5 +508,57 @@
     "material_thickness": 7,
     "flags": [ "STURDY", "OUTER", "RAINPROOF", "ONLY_ONE" ],
     "armor": [ { "encumbrance": 40, "coverage": 100, "covers": [ "torso", "arm_l", "arm_r" ] } ]
+  },
+  {
+    "id": "cuirass_chitin",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "chitinous cuirass" },
```
Comment (CountAlex) [inline data/json/items/armor/torso_armor.json:516]: This and next entries should have plural forms.

**Comment by I-am-Erk** [comment]
I am closing this as stale for now. Please feel free to reopen if you're back.

**Pull request closed without merging**